tex/chProofs.tex

```diff
@@ -920,10 +950,10 @@ \subsubsection{Case analysis with naming}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Rewriting}\label{ssec:rw}
 
-This section explains how to perform local replacements of certain
-subterms of a goal with other terms during the course of a formal
+This section explains how to locally replace certain
+subterms of a goal by other terms during the course of a formal
```

Comment:
"with" feels a bit better (please see this [english.SE question](http://english.stackexchange.com/questions/151309/replace-with-versus-replace-by))